helm_deploy/templates/deployment.yaml

```diff
@@ -11,6 +11,7 @@ spec:
   selector:
     matchLabels:
       {{- include "laa-submit-crime-forms.selectorLabels" . | nindent 6 }}
+      app: {{ include "laa-submit-crime-forms.fullname" . }}
```

Comment:
👍 especially as worker deployment is already doing this but with `-worker` appended

Reply:
Maybe suffix with web here as well?

Reply:
Agreed to leave this out.